doc/source/user_guide/io.rst

```diff
@@ -4858,6 +4859,20 @@ The above example creates a partitioned dataset that may look like:
    except OSError:
        pass
 
+.. _io.orc:
+
+ORC
+---
+
+.. versionadded:: 1.0.0
+
+Similar to the `parquet <io.parquet>` format, the `ORC Format <//https://orc.apache.org/>`__ binary columnar serialization
+for data frames. It is designed to make reading data frames efficient. Pandas provides *only* a reader for the
+ORC format, :func:`~pandas.read_orc`.
+
+See the documentation for `pyarrow <https://arrow.apache.org/docs/python/>`__ for more.
```

Comment:
There is actually no documentation about ORC there, so that link is not very helpful ...

Reply:
Can you mention this reader needs pyarrow to be installed instead?

Reply:
see 2 lines above about the ORC format

Reply:
docs are a bit sparse on the pyarrow side.

Reply:
created: https://issues.apache.org/jira/browse/ARROW-7296

Reply:
i was just looking through this PR to see who to ask about pyarrow.orc docs.

Reply:
iirc @jorisvandenbossche did a PR in arrow side for this